File: ios/GiphyReactNativeSDK.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(GiphyReactNativeSDK, NSObject)

RCT_EXTERN_METHOD(
    configure:(NSString)apiKey
)

RCT_EXTERN_METHOD(showGiphyView)

@end

File: ios/GiphyReactNativeSDK.swift
import GiphyUISDK

@objc(GiphyReactNativeSDK)
class GiphyReactNativeSDK: NSObject {
  @objc(configure:)
  func configure(config: NSDictionary) -> Void {
    if let apiKey = config["apiKey"] as? String {
      Giphy.configure(apiKey: apiKey)
    }
  }
}
